RIOT-OS/RIOT
# boards/nucleo-g070rb & g071rb: add pinouts to documentation

### Contribution description

This PR adds pinouts to the documentation pages for:
- `nucleo-g070rb`
- `nucleo-g071rb`

### Testing procedure

Generate doc and see if everything is fine.

```
make doc
xdg-open doc/doxygen/html/group__boards__nucleo-g070rb.html
xdg-open doc/doxygen/html/group__boards__nucleo-g071rb.html
```

### Issues/PRs references

None
